src/test/CheckResultsByLevel.test.tsx

```diff
@@ -0,0 +1,323 @@
+                "createdAt":"2023-05-11T20:47:53.869313Z",
+                "check":{
+                   "id":"3",
+                   "enableOn":null,
+                   "name":"Status: passed",
+                   "category":{
+                      "name":"Service Ownership"
+                   },
+                   "owner":null
+                },
+                "status":"passed"
+             }
+          ]
+       }
+    },
+    {
+       "level":{
+          "index":4,
+          "name":"very long level name"
+       },
+       "items":{
+          "nodes":[
+             {
+                "message":"This check has status upcoming_failed.",
+                "createdAt":"2023-05-11T20:47:53.869313Z",
+                "check":{
+                   "id":"4",
+                   "enableOn":"2023-05-11T20:47:53.869313Z",
+                   "name":"Status: upcoming_failed",
+                   "category":{
+                      "name":"Service Ownership"
+                   },
+                   "owner":null
+                },
+                "status":"failed"
+             },
+             {
+                "message":"This check has status upcoming_pending.",
+                "createdAt":"2023-05-11T20:47:53.869313Z",
+                "check":{
+                   "id":"5",
+                   "enableOn":"2023-05-11T20:47:53.869313Z",
+                   "name":"Status: upcoming_pending",
+                   "category":{
+                      "name":"Service Ownership"
+                   },
+                   "owner":null
+                },
+                "status":"pending"
+             },
+             {
+                "message":"This check has status upcoming_passed.",
+                "createdAt":"2023-05-11T20:47:53.869313Z",
+                "check":{
+                   "id":"6",
+                   "enableOn":"2023-05-11T20:47:53.869313Z",
+                   "name":"Status: upcoming_passed",
+                   "category":{
+                      "name":"Service Ownership"
+                   },
+                   "owner":null
+                },
+                "status":"passed"
+             }
+          ]
+       }
+    },
+    {
+       "level":{
+          "index":5,
+          "name":"and another one"
+       },
+       "items":{
+          "nodes":[
+             
+          ]
+       }
+    }
+  ];
+  const totalChecksData = 0;
+  const totalPassingChecksData = -1;
+
+  it('renders the appropriate check results', () => {
+    const wrapper = mount(
+      <CheckResultsByLevel
+        checkResultsByLevel={checkResultsByLevelData}
+        totalChecks={totalChecksData}
+        totalPassingChecks={totalPassingChecksData}
+      />
+    );
+
+    // Total checks header
+
+    const totalChecksCounter = wrapper.find("div.BackstageInfoCard-subheader-13");
+    expect(totalChecksCounter.length).toEqual(1);
+    expect(totalChecksCounter.at(0).text()).toEqual("Total Checks Passing: -1 / 0 (100%)")
```

Comment:
I get the feeling an AI wrote all these assertions... and this one in particular doesn't seems like something a human expects 😛 

Please consider splitting these up into different `it()` blocks and double check each is correct